manifestservice/manifests/__init__.py

```diff
@@ -320,6 +452,15 @@ def _generate_unique_manifest_filename(
     return filename
 
 
+def _generate_unique_metadata_filename(users_existing_metadata_files):
```

Comment:
This function looks exactly the same as `_generate_unique_manifest_filename()` above, I think we can consolidate them by given it a more generalized name

Reply:
Hi @mfshao I have addresses all your recent comments, thank you for the code review